### PR TITLE
Adds serac subdirectory

### PR DESCRIPTION
**Pull request by white238**
#282

### CODE REVIEW

**Comment by white238** [comment]
LGTM

File: src/serac/physics/CMakeLists.txt [hunk 1]
```diff
@@ -30,12 +30,7 @@ blt_add_library(
     DEPENDS_ON  mfem integrators numerics infrastructure physics_utilities physics_operators axom fmt mpi
     )
 
-target_include_directories(physics PUBLIC 
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/..>
-    $<INSTALL_INTERFACE:include>
-    )
```
Comment (joshessman-llnl) [inline src/serac/physics/CMakeLists.txt:36]: I might be missing something, but why was this removed?

Comment (white238) [reply src/serac/physics/CMakeLists.txt:36]: Everything in Serac inherits from `infrastructure` which has this PUBLIC.

**Review by jamiebramwell — APPROVED** [review]

**Comment by white238** [comment]
LGTM

**Review by joshessman-llnl — APPROVED** [review]

**Comment by samuelpmishLLNL** [comment]
Thanks for this PR, Chris-- it's going to save me a lot of headaches when working in the smith repo!

**Review by samuelpmishLLNL — APPROVED** [review]
Thanks

**Pull request merged**